packages/sources/alphavantage-test/src/config/index.ts

```diff
@@ -0,0 +1,16 @@
+export const DEFAULT_API_ENDPOINT = 'forex'
+export const customSettings = {
+  API_ENDPOINT: {
+    description: 'The HTTP URL to retrieve data from',
+    type: 'string',
+    default: 'https://www.alphavantage.co/query',
+    required: true,
```

Comment:
Since we have a default, we don't need to mark this as required

Reply:
Done